### DISCUSSION TITLE
AMS Filament and user profiles have separate color now

### DISCUSSION THREAD

**Discussion by DuckTown407** [OrcaSlicer/OrcaSlicer#686] [General]
In 1.5 when you had the AMS sync you would get the colors and filament types that you did on the AMS available in the slicer. Mine would always be a generic because I use brands that BL doesn't have profiles for. I would then set each one to a specific user profile that contains the real filament settings and the color would stay what I had set in the AMS. Now, in 1.6, all non AMS filaments  have a "default color" such that when you change an AMS filament to this type it overrides the color. Say I loaded in 2 PLA spools from the same brand and wanted  to use  those for a multicolor print: now I would need to have 2 different profiles, 1 for each color of that filament, rather than just selecting my user defined profile and assigning it to each AMS slot. Is this the intended behavior? Can there be a setting to make it the color of the AMS slot you chose?

**Comment by AznArt2023** [comment]
I just setup Orca 1.6 and noticed this too. This adds a lot of extra steps to my workflow as I need to go back and change all my colors. I base my settings on projects and filament brands, not specific colors. Will have to go back to orca 1.5 until this is resolved.
